src/components/NewCommentForm.tsx

```diff
@@ -1,103 +1,206 @@
-import React from 'react';
+import {
+  ChangeEvent,
+  Dispatch,
+  FC,
+  FormEvent,
+  SetStateAction,
+  useState,
+} from 'react';
+import cn from 'classnames';
+
+import { Comment } from '../types/Comment';
+import { addComment } from '../utils/api';
+
+const initialFormData = { name: '', email: '', body: '' };
+const initialFromErrors = { name: false, email: false, body: false };
+
+interface Props {
+  postId: number;
+  updateComments: Dispatch<SetStateAction<Comment[]>>;
+}
+
+export const NewCommentForm: FC<Props> = ({ postId, updateComments }) => {
+  const [formData, setFormData] = useState(initialFormData);
+  const [formErrors, setFormErrors] = useState(initialFromErrors);
+
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  const handleChangeField = (
+    event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+  ) => {
+    const { name, value } = event.target;
+
+    setFormData(prevData => ({ ...prevData, [name]: value.trimStart() }));
```

Comment:
There is a syntax error in this line. It seems like you intended to spread `prevData`, but the correct syntax is `{ ...prevData, [name]: value.trimStart() }`. The same issue is present in line 70 with `prevState`.